src/Orleans/AssemblyLoader/AssemblyLoader.cs

```diff
@@ -291,15 +81,15 @@ private static Assembly MatchWithLoadedAssembly(AssemblyName searchFor, IEnumera
                 var candidateFullName = assembly.FullName;
                 if (String.Equals(candidateFullName, searchForFullName, StringComparison.OrdinalIgnoreCase))
                 {
-                    return assembly;                    
+                    return assembly;
                 }
             }
             return null;
         }
 
         private static Assembly MatchWithLoadedAssembly(AssemblyName searchFor, AppDomain appDomain)
         {
-            return 
+            return
```

Comment:
@veikkoeeva  I think this PR is about removing dependency on AppDomain to move to support .Net Core, which is part of #2145 . That's why I get confused. 

About shimmed `AppDomain`, it might accelate our process on going to .Net Core. But it might also confuse people with AppDomain in .Net. I'm slightly leaning in the direction to not create that shimmed `AppDomain `class. :)